sites/kit.svelte.dev/package.json

```diff
@@ -3,7 +3,7 @@
 	"version": "0.0.1",
 	"scripts": {
 		"dev": "svelte-kit dev",
-		"build": "svelte-kit build",
+		"build": "svelte-kit build --verbose",
```

Comment:
Is this change intended? TBH I didn't even know there was a `--verbose` option to `svelte-kit build`.

Reply:
i prefer having it, because then i can see which pages are getting prerendered